proxy/src/main/java/org/apache/rocketmq/proxy/service/message/LocalRemotingCommand.java

```diff
@@ -17,15 +17,18 @@
 package org.apache.rocketmq.proxy.service.message;
 
 import java.util.HashMap;
+import org.apache.rocketmq.proxy.common.ProxyContext;
 import org.apache.rocketmq.remoting.CommandCustomHeader;
 import org.apache.rocketmq.remoting.exception.RemotingCommandException;
+import org.apache.rocketmq.remoting.protocol.LanguageCode;
 import org.apache.rocketmq.remoting.protocol.RemotingCommand;
 
 public class LocalRemotingCommand extends RemotingCommand {
 
-    public static LocalRemotingCommand createRequestCommand(int code, CommandCustomHeader customHeader) {
+    public static LocalRemotingCommand createRequestCommand(int code, CommandCustomHeader customHeader, ProxyContext ctx) {
```

Comment:
It's better to pass language rather than proxyContext into the constructor of LocalRemotingCommand.